src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -313,6 +313,8 @@ private function registerOAuthConfiguration(ContainerBuilder $container, array $
     private function registerSwaggerConfiguration(ContainerBuilder $container, array $config, XmlFileLoader $loader): void
     {
         if (empty($config['swagger']['versions'])) {
```

Comment:
`$container->setParameter('api_platform.swagger.versions', $config['swagger']['versions']);` could be added just before this line and removed everywhere else.